skytemple_files/graphics/chara_wan/sheets.py

```diff
@@ -831,8 +829,8 @@ def mapDuplicateImportImgs(imgs, final_imgs, img_map):
                     flip = final_idx
             else:
                 imgs_flip = exUtils.imgsEqual(final_img[0], img[0], True)
-                if imgs_flip:
-                    imgs_flip = exUtils.offsetsEqual(final_img[1], img[1], img[0].size[0], True)
+                # if imgs_flip:
```

Comment:
Can this be verified against multi-offset Pokemon like Jirachi?
Multi-offset Pokemon have multiple frames with the same textures but different offsets.  Will they still import correctly with this change?

Reply:
This should work with differing offsets, because the algorithm already have this system used for frames that are identical in texture but differing in offsets in the branch above that, I don't see why it should be restricted here.
As for Jirachi, there is indeed a problem with one of their offsets on Special2, but that problem is also present in the current version of the algorithm and may be linked with the animation frame for that offset being an empty frame.